### ISSUE TITLE
Ask:: Fail notification to use NotificationFailed Event.

### ISSUE THREAD

**Issue by setkyar** [benwilkins/laravel-fcm-notification#34]
Hi @benwilkins 

Did you have any plan to add `NotificationFailed` when notification got failed? If so, I would like to make a PR. 

Currently, when notification got failed. It was going to `failed_jobs`. Instead of that, by using `NotificationFailed` event, we can customize whatever we want. Like adding a `status` column to `notifications` table and customizing base on success and failure?

**Comment by setkyar** [comment]
@benwilkins You can see example usage on [other packages](https://github.com/laravel-notification-channels/twilio/blob/b662794cc0dea2cff882e505eb8cbeae44742b42/src/TwilioChannel.php#L60).

**Comment by poklin** [comment]
Do you guys have any guide for receiving the response? I keep getting a blank/null response but the notification is sent successfully

**Comment by setkyar** [comment]
@poklin I don't have any issue with receiving the notification response. Also, take a check [notification events on official docs](https://laravel.com/docs/5.8/notifications#notification-events).

**Comment by poklin** [comment]
@setkyar Yup, I did check and created Event and Listener. But I do not what is the "key word" to get the response back from FCMChannel. If I do dd() inspection on the FCMChannel, I did receive the response, just I do not know how to retrieve it from the front. >.<

**Comment by ankurk91** [comment]
@benwilkins 

I can send a PR for the `NotificationFailed`.
Laravel does not emit this event by default, package developer need to handle and dispatch it.
See [example](https://github.com/ankurk91/bandwidth-notification-channel/blob/a4547cca9c1d7b93465838dffcabe5ece246d392/src/BandwidthChannel.php#L102)
